angular-workspace/components/helpers.ts

```diff
@@ -0,0 +1,7 @@
+import { StoryFnAngularReturnType } from "@storybook/angular/dist/client/types";
+
+export function isStoryFnAngularReturnTypeTemplate(
```

Comment:
React biedt een check aan, voor Angular moesten we 'm ff zelf bakken.

Gelukkig heb je je toevallig in type guards ingelezen ;P